samples/boards/nrf52/nffs/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  description: TBD
+  name: TBD
+tests:
+  test:
+    build_only: true
+    platform_whitelist: qemu_cortex_m3 qemu_x86
```

Comment:
platform_whitelist: nrf52840_pca10056 nrf52_pca10040

In case you want to have quemu as well here (which is good Idea) you need to provide another proj.conf file for it, (like this one: https://github.com/zephyrproject-rtos/zephyr/blob/master/tests/subsys/fs/nffs_fs_api/prj.conf) and  ram beckend: https://github.com/zephyrproject-rtos/zephyr/blob/master/tests/subsys/fs/nffs_fs_api/src/test_ram_backend.c) - I gues you won't do that.